gensrc/thrift/DataSinks.thrift

```diff
@@ -59,6 +59,41 @@ enum TResultSinkType {
     VARIABLE
 }
 
+enum TParquetColumnType {
+    BOOLEAN,
+    INT32,
+    INT64,
+    INT96,
+    BYTE_ARRAY,
+    FLOAT,
+    DOUBLE,
+    FIXED_LEN_BYTE_ARRAY,
+}
+
+enum TParquetRepetitionType {
+    REQUIRED,
+    OPTIONAL,
+    REPEATED,
+}
+
+struct TParquetColumn {
+    1: required string name;
+    2: optional TParquetColumnType type;
+    3: optional TParquetRepetitionType repetition_type
+    4: optional i64 id;
+}
+
+struct TParquetSchema {
```

Comment:
Schema is a common information. If we want to support write in ORC format, we can't reuse the schema. Schema translation in BE is more appropriate.